apps/modernization-ui/src/apps/page-builder/pages/ManagePages/ManagePagesTable.tsx

```diff
@@ -41,6 +41,8 @@ export const ManagePagesTable = ({ summaries, currentPage, pageSize, totalElemen
     const [tableRows, setTableRows] = useState<TableBody[]>([]);
     const { searchQuery, setSearchQuery, setCurrentPage, setSortBy, setSortDirection } = useContext(PagesContext);
     const { state } = useContext(UserContext);
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    const [searchParams, setSearchParams] = useSearchParams();
```

Comment:
Question: I don't really like disabling eslint. Does `const setSearchParams = useSearchParams()[1]` work?  Or is there a more elegant solution

Reply:
Unfortunately that doesn't work. I can do this though...
```
const searchParams = useSearchParams();
const setSearchParams = searchParams[1];
```